srcpkgs/zfs/files/zed/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . conf
+exec zed -F $OPTS
```

Comment:
Is this a recommended service?

Reply:
It's useful if you care about receiving notifications for ZFS events, but I'm not sure I'd call it "recommended".

Reply:
Fair enough. Just wondering about the docs side of it c:

Thanks